src/encoded/schemas/changelogs/file.md

```diff
@@ -1,5 +1,11 @@
 ## Changelog for file.json
 
+### Schema version 15
```

Comment:
@yunhai I think you need to also include the new types that have been added, not only the ones that were modified

Reply:
and preferred_default also